scp/assets/data/faqData.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "question": "What are the circumstances under which a student is shown the door and asked to leave the campus ?",
+    "answer": "As per the Academic Regulations \n(Link: https://drive.google.com/open?id=1le3h9A7CZcCAGScwAxRrMxrKJD5Kfmq1 ) and \nDisciplinary manual \n(Link: https://drive.google.com/open?id=1Ya5MpIJbAnXZ-sZK-X5UCg-0en03X13z )\nOne student must leave if:\n• If a student gets a CGPA less than 6.00 in three consecutive semesters, he must leave the Institute.\n• After first year if the CGPA < 5.00\n• As per the clause P-24 and P-25 of disciplinary manual, outright expulsion from the institute can also be a step under certain circumstances"
+  },
+  {
+    "question": "Am I a superhero I think I am 2?",
+    "answer": "There are some possibilities you are a superhero."
```

Comment:
You are a superhero 🚀

Reply:
Haha thats a tempory json file 😂 I will update it asap